stories/Button/Button.regression-test.story.jsx

```diff
@@ -3,17 +3,73 @@ import { storiesOf } from '@storybook/react';
 import Button from '../../components/Button';
 
 storiesOf('Buttons', module)
-  .add('default', () => <Button>Catho</Button>)
-  .add('disabled', () => <Button disabled>Catho</Button>)
-  .add('stroked', () => <Button stroked>Catho</Button>)
-  .add('full', () => <Button full>Catho</Button>)
-  .add('icon info', () => <Button icon="info">Catho</Button>)
-  .add('size xsmall', () => <Button size="xsmall">Catho</Button>)
-  .add('size small', () => <Button size="small">Catho</Button>)
-  .add('size large', () => <Button size="large">Catho</Button>)
-  .add('size xlarge', () => <Button size="xlarge">Catho</Button>)
-  .add('skin neutral', () => <Button skin="neutral">Catho</Button>)
-  .add('skin secondary', () => <Button skin="secondary">Catho</Button>)
-  .add('skin success', () => <Button skin="success">Catho</Button>)
-  .add('skin warning', () => <Button skin="warning">Catho</Button>)
-  .add('skin error', () => <Button skin="error">Catho</Button>);
+  .add('primary default', () => <Button>Catho</Button>)
+  .add('primary disabled', () => <Button disabled>Catho</Button>)
+  .add('primary stroked', () => <Button stroked>Catho</Button>)
+
+  .add('primary full', () => <Button full>Catho</Button>)
+  .add('primary icon info', () => <Button icon="info">Catho</Button>)
+  .add('primary size xsmall', () => <Button size="xsmall">Catho</Button>)
+  .add('primary size small', () => <Button size="small">Catho</Button>)
+  .add('primary size large', () => <Button size="large">Catho</Button>)
+  .add('primary size xlarge', () => <Button size="xlarge">Catho</Button>)
+
+  .add('secondary default', () => <Button skin="secondary">Catho</Button>)
+  .add('secondary disabled', () => (
```

Comment:
how about to keep the pattern in one line on tests, if possible?
Just link the primary?
was this made by linter?

Reply:
I tried to keep them in one line but linter made these changes.